src/test/java/LoginTests.java

```diff
@@ -1,25 +1,69 @@
+import Pages.HomePage;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

For example `pages`
Rename refactorings https://www.jetbrains.com/help/idea/rename-refactorings.html#rename_class_example

![image](https://github.com/user-attachments/assets/fd5705aa-19b2-4a39-bcd2-f93917512b7c)